_posts/wookiemage/2016-2-3-Colins-Chapter-5.md

```diff
@@ -0,0 +1,13 @@
+---
+author: wookiemage
+layout: post
+title: "Colin's Chapter 5 Exercises
+"
+---
+  Here's the first exercise:
+  <iframe src="https://trinket.io/embed/python/39cbbd5989" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+  Here's the second:
+  <iframe src="https://trinket.io/embed/python/f584d23e33" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
In this one there's an error in your min logic if the user's input is all negative.  You probably need the same setup you had with max.